android/app/src/main/java/com/woowacourse/ody/presentation/creation/MeetingCreationViewModel.kt

```diff
@@ -72,14 +63,23 @@ class MeetingCreationViewModel : ViewModel() {
         }
     }
 
-    fun emptyNickname() {
-        nickname.value = ""
+    fun initializeMeetingTime() {
+        if (meetingHour.value != null || meetingMinute.value != null) {
+            return
+        }
```

Comment:
`if (!(meetingHour.value == null && meetingMinute.value == null))`은 어떤가요?

Reply:
```suggestion
    fun initializeMeetingTime() {
        if (meetingHour.value == null || meetingMinute.value == null) {
            return
        }
```
헐 근데 해음 덕분에 깨달았는데 제가 로직 자체를 실수한 것 같아요. ㅋㅋ 위 코드로 변경된다면 수정하지 않아도 괜찮을까요?

Reply:
괄호가 또 들어가면 로직을 이해하기 위해 한 번 더 생각해봐야 한다고 느껴져서요. 사실 그냥 취향 차이인 것 같습니다...

Reply:
변수로 분리했습니다!!! [4f81c3b](https://github.com/woowacourse-teams/2024-ody/pull/189/commits/4f81c3b0fbd651d0bb0bc31c53ca6b2d1a744bc4)